ee/tabby-webserver/src/service/event_logger.rs

```diff
@@ -15,12 +18,16 @@ fn log_err<T, E: Display>(res: Result<T, E>) {
 }
 
 pub fn new_event_logger(db: DbConn) -> impl EventLogger {
```

Comment:
```suggestion
pub fn new_event_logger(db: DbConn) -> impl RawEventLogger {
```

Just realized you implement RawEventLogger directly for EventLogger

Reply:
Yes, that was why I had added the supertrait relationship - `RawEventLogger` is implemented with a blanket impl for all `EventLogger`